certified-connectors/Tikit/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2652 @@
+            "x-ms-summary": "$skip"
+          },
+          {
+            "name": "$count",
+            "in": "query",
+            "description": "Define oData count",
+            "type": "boolean",
+            "x-ms-visibility": "advanced",
+            "x-ms-summary": "$count"
+          }
+        ]
+      }
+    },
+    "/AddTicket": {
+      "post": {
+        "summary": "Create New Ticket",
+        "description": "Create New Ticket",
+        "operationId": "CreateTicket",
+        "parameters": [
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServiceDesk.Core.Actions.AddTicket"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response from AddTicket",
+            "schema": {
+              "$ref": "#/definitions/ServiceDesk.Core.Models.Ticket"
+            }
+          },
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        }
+      }
+    },
+    "/UpdateTicket": {
+      "post": {
+        "summary": "Update a Ticket",
+        "description": "Update a Ticket",
+        "operationId": "UpdateTicket",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServiceDesk.Core.Actions.UpdateTicket"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response from AddTicket",
+            "schema": {
+              "$ref": "#/definitions/ServiceDesk.Core.Models.Ticket"
+            }
+          },
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        }
+      }
+    },
+    "/ticket/{id}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "RequesterId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "RequesterId"
+                },
+                "Title": {
+                  "type": "string",
+                  "description": "Title"
+                },
+                "Closed": {
+                  "type": "boolean",
+                  "description": "Closed"
+                },
+                "DueDate": {
+                  "type": "string",
+                  "description": "DueDate"
+                },
+                "ResolutionDate": {
+                  "type": "string",
+                  "description": "ResolutionDate"

```

Comment:
@TikitTeam All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
updated